lib/plugin.js

```diff
@@ -1,5 +1,7 @@
 const _layer = '<%= options.layer %>'
 const _id = '<%= options.id %>'
+// Remove trailing slash to avoid duplicate slashes when appending route path
+const _routerBase = '<%= options.routerBase.replace(/\/+$/, '') %>'
```

Comment:
Should we consider [trailingSlash](https://github.com/nuxt/nuxt.js/pull/6331) here? 

Also, I think we can simply do normalization from module. (probably line before return)

Reply:
As we discussed online, the `trailingSlash` option should not affect this logic.
There is a small issue with it that is unrelated to this PR though. I'll create an issue for it.